redpanda/values.yaml

```diff
@@ -218,42 +286,127 @@ serviceAccount:
+  # kafka_group_recovery_timeout_ms: 30000ms                     # Kafka group recovery timeout expressed in milliseconds
+  # kafka_qdc_depth_alpha: 0.8                                   # Smoothing factor for kafka queue depth control depth tracking
+  # kafka_qdc_depth_update_ms: 7s                                # Update frequency for kafka queue depth control
+  # kafka_qdc_enable: false                                      # Enable kafka queue depth control
+  # kafka_qdc_idle_depth: 10                                     # Queue depth when idleness is detected in kafka queue depth control
+  # kafka_qdc_latency_alpha: 0.002                               # Smoothing parameter for kafka queue depth control latency tracking
+  # kafka_qdc_max_depth: 100                                     # Maximum queue depth used in kafka queue depth control
+  # kafka_qdc_max_latency_ms: 80ms                               # Max latency threshold for kafka queue depth control depth tracking
+  # kafka_qdc_min_depth: 1                                       # Minimum queue depth used in kafka queue depth control
+  # kafka_qdc_window_count: 12                                   # Number of windows used in kafka queue depth control latency tracking
+  # kafka_qdc_window_size_ms: 1500ms                             # Window size for kafka queue depth control latency tracking
+  # kvstore_flush_interval: 10ms                                 # Key-value store flush interval (ms)
+  # kvstore_max_segment_size: 16MB                               # Key-value maximum segment size (bytes)
+  # log_cleanup_policy: deletion                                 # Default topic cleanup policy
+  # log_compaction_interval_ms: 5min                             # How often do we trigger background compaction
+  # log_compression_type: producer                               # Default topic compression type
+  # log_message_timestamp_type: create_time                      # Default topic messages timestamp type
+  # log_segment_size: 1GB                                        # How large in bytes should each log segment be (default 1G)
+  # max_compacted_log_segment_size: 5GB                          # Max compacted segment size after consolidation
+  # max_kafka_throttle_delay_ms: 60000ms                         # Fail-safe maximum throttle delay on kafka requests
+  # metadata_dissemination_interval_ms: 3000ms                   # Interaval for metadata dissemination batching
+  # metadata_dissemination_retries: 10                           # Number of attempts of looking up a topic's meta data like shard before failing a request
+  # metadata_dissemination_retry_delay_ms: 500ms                 # Delay before retry a topic lookup in a shard or other meta tables
+  # quota_manager_gc_sec: 30000ms                                # Quota manager GC frequency in milliseconds
+  # rack: None                                                   # Rack identifier
```

Comment:
Let's open a ticket to support rack awareness when installed across multiple AZs.